geonode/base/management/commands/backup.py

```diff
@@ -17,366 +17,14 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 #########################################################################
+from geonode import geoserver, qgis_server
+from geonode.utils import check_ogc_backend
 
-try:
-    import json
-except ImportError:
-    from django.utils import simplejson as json
-import os
-import time
-import shutil
-import requests
-import re
-import helpers
-from helpers import Config
+if check_ogc_backend(geoserver.BACKEND_PACKAGE):
```

Comment:
Basically check backend and load appropriate class Command based on what backend is currently being used.